editor/src/uuiui/icons.tsx

```diff
@@ -307,6 +307,12 @@ export const Icons = {
     width: 18,
     height: 18,
   }),
+  ComponentInstance: makeIcon({
+    category: 'element',
+    type: 'componentinstance',
+    width: 18,
+    height: 18,
+  }),
```

Comment:
Looks like this was committed accidentally

Reply:
oh yeah, wanted to pull that to another branch but it's just an asset. Will update the PR